packages/libs/react-ui/src/components/Link/Link.tsx

```diff
@@ -7,21 +7,21 @@ import type { FC, ReactNode } from 'react';
 import React from 'react';
 
 export interface ILinkProps {
-  href?: string;
-  target?: '_blank' | '_self' | '_parent' | '_top';
+  asChild?: boolean;
+  block?: boolean;
```

Comment:
I see this was added before, but this is actually inconsistent with how we apply styles. I've made a ticket for it: https://app.asana.com/0/1205693948735844/1205687963086439

Reply:
I think I only re-ordered this list. Unrelated to this PR otherwise.